.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Docker image
+
+on:
+  push:
+    paths-ignore:
+      - README.md
+      - ansible/*
+      - .vscode/*
+      - .venv/*
+      - .pytest_cache/*
+      - '**__pycache__/*'
+      - tests/.pytest_cache/*
+    branches:
+      - master
+      - exercise-8 # TODO exercise-8: Remove once have tested - only want to create image when master updated
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      
+      - name: Build prod docker image
+        run: docker build --target production --tag ${{ secrets.DOCKER_USERNAME }}/todo-app:prod .
+      
+      - name: Push prod docker image to Docker Hub
+        run: docker push ${{ secrets.DOCKER_USERNAME }}/todo-app:prod
```

Comment:
Note that you've already completed 90% of the CD pipeline work here in addition to the cloud deployment - all that's left is to add the webhook call to this pipeline 😄